docs/guides/mock-responses.md

```diff
@@ -0,0 +1,159 @@
+        schema:
+          "$ref": "#/components/schemas/Pet"
+        examples:
+          cat:
+            summary: An example of a cat
+            value:
+              id: 2
+              name: Fluffy
+              photoUrls: []
+          dog:
+            summary: An example of a dog
+            value:
+              id: 1
+              name: Spot
+              photoUrls: 
+              - https://images.dog.ceo/breeds/kelpie/n02105412_893.jpg
+```
+
+Calling `curl http://127.0.0.1:4010/pets/123` on this will give:
+
+```json
+{
+    "id": 2,
+    "name": "Fluffy",
+    "photoUrls": []
+}
+```
+
+Changing the URL to `http://127.0.0.1:4010/pets/123?__example=dog`
+
+```json
+{
+    "id": 2,
+    "name": "Spot",
+    "photoUrls": [
+      "https://images.dog.ceo/breeds/kelpie/n02105412_893.jpg"
+    ]
+}
+```
+
+## Static or Dynamic Generation
+
+By default, Prism will use a **static generation strategy**, which is outlined below. You can enable the dynamic examples generation by using the `-d` flag in the command line.
+
+```yaml
+prism mock api.oas3.yaml # Static examples
+
+prism mock -d api.oas3.yaml # Dynamic examples
+```
+
+If the HTTP server has been started in static mode, specific calls can be made in dynamic mode by appending `?__dynamic=true` to any URL.
+
+### Static Response Generation
+
+If the provided OpenAPI Schema Object has a response body example then it'll use that whole thing.
+
+If not, an response body will be created by looking through the whole `schema` object (following any `$ref`'s it finds along the way) to create a full fake response.
+
+* If the property has a default value, then it will return the specified value
+* If the property has an `examples` value, then it will return the first element in the array
+* If the property has nor an example or a default value and it's **nullable**, it will return null
+* If the property has nor an example or a default value and it's **not** nullable, and has a `format` specified it will return a meaningful static value according to the format
+* If the property has nor an example or a default value and it's **not** nullable, and has not a `format` specified it will return `'string'` in case of a number and `0` in case of a string
+
+Let's try an example! 🐶
+
+This is a schema component found in our OpenAPI description document:
+
+```yaml
+  Pet:
+    type: object
+    properties: 
+      id: 
+        type: integer
+        format: int64
+      name: 
+        type: string
+        example: doggie
+      photoUrls: 
+        type: array
+        items: 
+          type: string
+```
+
+When we call `curl http://127.0.0.1:4010/pets/123`, the operation references this component so we get a doggie:
+
+```json
+{
+    "id": 0,
+    "name": "doggie",
+    "photoUrls": [
+        "string"
+    ]
+}
+```
+
+Notice that `name` had an `example` with a value so Prism used it, but `photoUrls` did not, so it just returned `"string"`. 🤷‍♂️
+
+### Dynamic Response Generation
```

Comment:
Not rendering correctly

<img width="531" alt="image" src="https://user-images.githubusercontent.com/2294309/62794065-c8e8b500-ba98-11e9-937c-ef7346ef0051.png">

Reply:
Fixed! Great spot, this is a really weird whitespace bug which I'll make an issue for.